src/components/Comments/CommentCard.tsx

```diff
@@ -100,16 +100,18 @@ const CommentCard = ({
                             rows={5}
                             id="NewPunchComment"
                             onChange={handleCommentChange}
-                        />
-                        <Button onClick={buttonClick}>Add comment</Button>{' '}
+                        />{' '}
```

Comment:
Test a slight margin/space between the button and the text field, I think it would look better. 8 px should be enough I think